logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -99,14 +101,25 @@ func (lm *Plugin) Init() error {
 			}
 		}
 
-		// try to set log levels (note, not all of them might exist yet)
-		for _, cfgLogger := range lm.Conf.Loggers {
-			if err := lm.LogRegistry.SetLevel(cfgLogger.Name, cfgLogger.Level); err != nil {
-				//intentionally just log warn & not propagate the error (it is minor thing to interrupt startup)
-				lm.Log.Warn("setting level failed:", err)
+		// Update log levels for all loggers which already exist
+		loopAllLoggers:
+		for loggerName := range lm.LogRegistry.ListLoggers() {
```

Comment:
This will list only loggers from the registry that were already created before this point. If any logger is created after `Init` or in `AfterInit` it won't have the level changed.